src/app/about/about.component.ts

```diff
@@ -10,5 +10,6 @@ import { Component } from '@angular/core';
     templateUrl: './about.component.html'
 })
 export class AboutComponent {
+    open: Boolean = false;
```

Comment:
The `open` property controls the hidden/visible status of the  dummy `<clr-modal>` in `about.component.html` 

If not declared/initialised in the component, on AOT build it throws the following error.
![image](https://cloud.githubusercontent.com/assets/11031124/25441412/cd3fbc18-2abf-11e7-958e-e64cfd9c29a1.png)